hello-world-notification-microservice/README.md

```diff
@@ -72,4 +76,69 @@ we will receive the following message as we have specified to transform the filt
   "source": {"id":"<DEVICE ID>"},
   "type": "c8y_Speed"
 }
-```
\ No newline at end of file
+```
+
+
+## Prerequisite
+- cumulocity tenant with microservice hosting feature enabled
+  > Important: The Microservice hosting feature must be activated on your tenant, otherwise your request will return an error message like “security/Forbidden, access is denied”. This feature is not assigned to tenants by default, so trial accounts won’t have it. Contact [product support](https://cumulocity.com/guides/welcome/contacting-support/) so that we can assist you with the activation. Note that this is a paid feature. Microservice SDK for Java - Cumulocity IoT Guides
+- contents of this repository
+
+### Instructions
+1. Creating a test user
```

Comment:
Create a test user in the Cumulocity Management UI

Reply:
I made the change... if I understood you correctly

Reply:
fixed so it reflects this